sdk/script/management-sdk.ts

```diff
@@ -332,7 +332,7 @@ class AccountManager {
                         }
 
                         if (res.ok) {
-                            resolve(<void>null);
+                            resolve(<Package>(JSON.parse(res.text).package));
```

Comment:
I believe we need to wrap up `JSON.parse` into the `try/catch`, just in case. (please see the same approach below https://github.com/Microsoft/code-push/pull/533/files#diff-84f033a739bf58615844d4686a717544R337)